spec/views/curation_concerns/file_set/show.html.erb_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+describe 'curation_concerns/file_sets/show.html.erb', type: :view do
+  before do
+    allow(view).to receive(:parent).and_return(parent)
+    view.lookup_context.prefixes.push 'curation_concerns/base'
+    allow(view).to receive(:can?).with(:edit, SolrDocument).and_return(true)
+    allow(view).to receive(:can?).with(:edit, String).and_return(true)
+    assign(:presenter, presenter)
+    assign(:document, solr_doc)
+  end
+
+  let(:parent) { stub_model(GenericWork) }
+  let(:user) { double(user_key: 'sarah', twitter_handle: 'test') }
+  let(:file_set) do
+    create(FileSet, id: '123',
```

Comment:
I'd prefer that we don't create models in view specs.  Can you just produce the values that go into solr without hitting Fedora?